cmd/discovery.go

```diff
@@ -82,6 +95,7 @@ func discoverNodesForService(serviceName string) ([]esnode, error) {
 			name:    svc.Node,
 			ip:      svc.Address,
 			port:    svc.ServicePort,
+			scheme:  getScheme(svc),
```

Comment:
OPT: can we keep same naming than clusterNameFromTags with same signature (schemeFromTags(svc.ServiceTags))

Reply:
+1

Reply:
Done